core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -932,9 +938,43 @@ public static String getDefaultOncoprintView() {
         }
         return defaultOncoprintView.trim();
     }
-
+    
+    public static boolean useCustomDriverAnnotations() {
+        String useCustomDriverAnnotations = properties.getProperty(USE_CUSTOM_DRIVER_ANNOTATIONS);
```

Comment:
yes, if omitted from portal.properties, it means "true" by default and the logic below will set it to false if it is not applicable. You can still have it in portal.properties.EXAMPLE (which has become a kind of place where the properties are documented, but have it commented out there. Something like:
```
# set this to false to disable the automatic selection of the custom driver and passenger mutations annotation in the oncoprint (instead of OncoKB and Hotspots). By default this is true. Setting this to false will select OncoKB and Hotspots as the default annotation source instead.
#oncoprint.custom_driver_annotation.default=false
```